starter-code/js/index.js

```diff
@@ -2,8 +2,9 @@ function deleteItem(e){
 
 }
 
-function getPriceByProduct(itemNode){
 
+function getPriceByProduct(itemNode){
+  return itemNode[0].innerText;
```

Comment:
This will only return the price of the first of the products found by the document.getElementsByClassName("item-price");